Library/Homebrew/vendor/bundle/bundler/setup.rb

```diff
@@ -11,45 +11,48 @@
 $:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/zeitwerk-2.3.0/lib"
 $:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/activesupport-6.0.2.2/lib"
 $:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/ast-2.4.0/lib"
+$:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/bindata-2.4.6/lib"
 $:.unshift "#{path}/"
 $:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/connection_pool-2.2.2/lib"
-$:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/extensions/universal-darwin-19/2.6.0/json-2.3.0"
```

Comment:
Some of the path changes here indicate that we might need to split the `setup.rb` between Linux and macOS -- I'm guessing that changing from `universal-darwin-19` to `x86_64-linux` here and below is going to make loading these on macOS fail 🙂

Reply:
@woodruffw because these are native extensions for things that aren't vendored it's actually fine: it'll just end up fixing them on Linux when you e.g. `brew audit`. It'd be better if these were macOS paths but I can fix that easily enough.

Reply:
> it'll just end up fixing them on Linux when you e.g. `brew audit`. It'd be better if these were macOS paths but I can fix that easily enough.

Good to know!